cli/cmd/install.go

```diff
@@ -254,7 +255,7 @@ func isRunAsRoot(values map[string]interface{}) bool {
 // renderChartToBuffer takes a slice of loaded template files and configuration values and renders
 // them into a buffer. The coalesced values are also returned so that they may be rendered via
 // `renderOverrides` if appropriate.
-func renderChartToBuffer(files []*loader.BufferedFile, values *l5dcharts.Values, valuesOverrides map[string]interface{}) (*bytes.Buffer, chartutil.Values, error) {
```

Comment:
I'm curious about why we aren't using the `l5dcharts.Values` going forward

Reply:
We always need to convert the Values struct into a map so that we can pass it to the Helm rendering library.  In this change, we're moving that conversion earlier so that we can reuse the `renderChartToBuffer` function between the control plane chart rendering path and the CRD chart rendering path since the CRD chart does not have a Values struct.